demisto_sdk/scripts/generate_commands_docs/generate_commands_docs.py

```diff
@@ -0,0 +1,131 @@
+import inspect
+import sys
+from pathlib import Path
+from typing import List
+
+from typer.main import get_command
+
+from demisto_sdk.__main__ import app
+
+
+def extract_changed_commands(modified_files: List[str]) -> List[str]:
+    """Extract the command names from the list of modified files."""
+    changed_commands = []
+    for file in modified_files:
+        # Check if the modified file ends with '_setup.py'
+        if file.endswith("_setup.py"):
+            command_name = Path(file).stem.replace("_setup", "")
+            changed_commands.append(command_name)
+    return changed_commands
+
+
+def get_sdk_command(command_name: str):
+    click_app = get_command(app)
+    command = click_app.commands.get(command_name)  # type: ignore[attr-defined]
+
+    if command is None:
+        return f"No README found for command: {command_name}"
+    return command
+
+
+def get_command_description(command_name: str) -> str:
+    """Retrieve the description (docstring) for the command."""
+    command = get_sdk_command(command_name)
+    if isinstance(command, str):
+        return command
+
+    command_func = command.callback
+    return inspect.getdoc(command_func) or "No description provided"
+
+
+def get_command_options(command_name: str) -> str:
+    """Generate the options section for the command."""
+    command = get_sdk_command(command_name)
+    if isinstance(command, str):
+        return command
+    options_text = "### Options\n\n"
+    for param in command.params:
+        param_name = (
+            f"--{param.name.replace('_', '-')}"
+            if param.param_type_name == "option"
+            else param.name
+        )
+        options_text += (
+            f"- **{param_name}**: {param.help or 'No description provided'}\n"
+        )
+        if param.default is not None:
```

Comment:
why `is not None:`? Isn't `if param.default` enough? if for example it is`''` you still don't want the condition to apply right?